parsers/src/test/scala/uk/gov/ons/addressIndex/parsers/FeatureAnalysersTest.scala

```diff
@@ -0,0 +1,334 @@
+    val input = "hll"
+    val expected = false
+    val actual = hasVowelsAnalyser.apply(input)
+    expected shouldBe actual
+  }
+
+  it should "produce the correct `has.vowels` feature output (true) for expected input (Surbiton)" in {
+    val input = "Surbiton"
+    val expected = true
+    val actual = hasVowelsAnalyser.apply(input)
+    expected shouldBe actual
+  }
+
+  it should "produce the correct `flat` feature output for expected inputs" in {
+    val inputs = Seq(
+      "FLAT", "FLT",
+      "APARTMENT", "APPTS", "APPT", "APTS", "APT",
+      "ROOM",
+      "ANNEX",  "ANNEXE",
+      "UNIT",
+      "BLOCK", "BLK"
+    )
+    val expected = Seq.fill(inputs.length)(true)
+    val actual = inputs map flatAnalyser.apply
+    expected should contain theSameElementsAs actual
+  }
+
+  it should "produce the correct `company` feature output for expected inputs" in {
+    val inputs = Seq(
+      "CIC",
+      "CIO",
+      "LLP",
+      "LP",
+      "LTD",
+      "LIMITED",
+      "CYF",
+      "PLC",
+      "CCC",
+      "UNLTD",
+      "ULTD"
+    )
+    val expected = Seq.fill(inputs.length)(true)
+    val actual = inputs map companyAnalyser.apply
+    expected should contain theSameElementsAs actual
+  }
+
+  it should "produce the correct `road` feature output for expected inputs" in {
+    val inputs = Seq(
+      "ROAD", "RAOD", "RD",
+      "DRIVE", "DR",
+      "STREET", "STRT",
+      "AVENUE", "AVENEU",
+      "SQUARE",
+      "LANE", "LNE", "LN",
+      "COURT", "CRT", "CT",
+      "PARK", "PK",
+      "GRDN", "GARDEN",
+      "CRESCENT",
+      "CLOSE", "CL",
+      "WALK",
+      "WAY",
+      "TERRACE",
+      "BVLD",
+      "HEOL",
+      "FFORDD",
+      "PLACE",
+      "GARDENS",
+      "GROVE",
+      "VIEW",
+      "HILL",
+      "GREEN"
+    )
+    val expected = Seq.fill(inputs.length)(true)
+    val actual = inputs map roadAnalyser.apply
+    expected should contain theSameElementsAs actual
+  }
+
+  it should "produce the correct `residential` feature output for expected inputs" in {
+    val inputs = Seq(
+      "HOUSE", "HSE",
+      "FARM",
+      "LODGE",
+      "COTTAGE", "COTTAGES",
+      "VILLA", "VILLAS",
+      "MAISONETTE",
+      "MEWS"
+    )
+    val expected = Seq.fill(inputs.length)(true)
+    val actual = inputs map residentialAnalyser.apply
+    expected should contain theSameElementsAs actual
+  }
+
+  it should "produce the correct `business` feature output for expected inputs" in {
+    val inputs = Seq(
+      "OFFICE",
+      "HOSPITAL",
+      "CARE",
+      "CLUB",
+      "BANK",
+      "BAR",
+      "UK",
+      "SOCIETY",
+      "PRISON",
+      "HMP",
+      "RC",
+      "UWE",
+      "UEA",
+      "LSE",
+      "KCL",
+      "UCL",
+      "UNI", "UNIV", "UNIVERSITY", "UNIVERISTY"
+    )
+    val expected = Seq.fill(inputs.length)(true)
+    val actual = inputs map businessAnalyser.apply
+    expected should contain theSameElementsAs actual
```

Comment:
For future reference, you're comparing expected with a List filled with `true` so ordering is not in question